static/structured/upcomingevents.json

```diff
@@ -614,5 +614,16 @@
 		"smallbonus": {
 			"traits": ["jake","data","uhura"]
 		}
+	},
+	{
+		"name": "A Good Lead",
+		"type": "Galaxy",
+		"dates": "Thursday 06/03 until Monday 06/07 at Noon (16.00 UTC)",
+		"startDate": "2021-06-03T16:00:00Z",
+		"endDate": "2021-06-07T16:00:00Z",
+		"highbonus": ["zyree_crew", "garak_evsuit_crew", "tucker_evsuit_crew", "quark_tribble_crew],
```

Comment:
```suggestion
		"highbonus": ["zyree_crew", "garak_evsuit_crew", "tucker_evsuit_crew", "quark_tribble_crew"],
```

Reply:
ugh, thank you for picking that up @ineffyble :)